docs/minio-admin-complete-guide.md

```diff
@@ -101,6 +101,22 @@ Alias is simply a short name to your MinIO service. MinIO end-point, access and
 mc config host add minio http://192.168.1.51:9000 BKIKJAA5BMMU2RHO6IBB V7f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12
 ```
 
+Host can also be added by passing access key , secret key using prompt
+
+```sh
+mc config host add <ALIAS> <YOUR-MINIO-ENDPOINT>
+```
+
+This asks for details
```

Comment:
Make this default behavior and positional as optional.. meaning move the preference of prompt based input as primary style.

Also the wording can be improved @eco-minio please help here .